diff -urN a/uxplay.cpp b/uxplay.cpp
--- a/uxplay.cpp	2023-11-06 20:49:36.000000000 +0800
+++ b/uxplay.cpp	2023-11-20 08:59:57.838512250 +0800
@@ -459,7 +459,7 @@
     printf("          another choice when using v4l2h264dec: v4l2convert\n");
     printf("-vs ...   Choose the GStreamer videosink; default \"autovideosink\"\n");
     printf("          some choices: ximagesink,xvimagesink,vaapisink,glimagesink,\n");
-    printf("          gtksink,waylandsink,osxvideosink,kmssink,d3d11videosink etc.\n");
+    printf("          gtksink,waylandsink,osxvideosink,kmssink,d3d11videosink,nufbdevsink etc.\n");
     printf("-vs 0     Streamed audio only, with no video display window\n");
     printf("-v4l2     Use Video4Linux2 for GPU hardware h264 decoding\n");
     printf("-bt709    A workaround (bt709 color) sometimes needed on RPi\n"); 
@@ -1593,6 +1593,10 @@
                "Use Alt-Enter key combination to toggle into/out of full-screen mode\n");
     }
 
+    if (videosink == "nufbdevsink" && use_video) {
+        videosink.append(" ! fakesink");
+    }
+
     if (bt709_fix && use_video) {
         video_parser.append(" ! ");
         video_parser.append(BT709_FIX);
